tests/modules/get_all_actions_by_ra/app/test_get_all_actions_by_ra_viewmodel.py

```diff
@@ -0,0 +1,90 @@
+from src.modules.get_all_actions_by_ra.app.get_all_actions_by_ra_viewmodel import GetAllActionsByRaViewmodel
+from src.shared.infra.repositories.action_repository_mock import ActionRepositoryMock
+from src.modules.get_all_actions_by_ra.app.get_all_actions_by_ra_usecase import GetAllActionsByRaUsecase
+
+class Test_GetAllActionsByRaViewmodel:
+    def test_get_all_actions_by_ra_viewmodel(self):
+        repo = ActionRepositoryMock()
+        usecase = GetAllActionsByRaUsecase(repo)
+        actions = usecase(ra='21017310')
+        viewmodel = GetAllActionsByRaViewmodel(actions)
+        expected = {
+   "actions":[
+      { 
+         "owner_ra":"21017310",
+         "date":"2021-10-18T00:00:00",
+         "action_id":"u1e2",
+         "title":"Reunião de Diretoria",
+         "duration":"02:00:00",
+         "project_code":"MF",
+         "associated_members_ra":[
```

Comment:
colocar ação como se fosse única do cara (retirar parâmetro associated_members_ra e owner_ra, colocando um parâmetro ra e um parâmetro is_owner bool true se ele for owner)